common/zoom/zoom-scheduled-meeting.ts

```
@@ -94,13 +89,15 @@ const createZoomMeeting = async (zoomUsers: ZoomUser[], startTime: Date, isCours
     if (response.status === 201) {
         logger.info(`Zoom - The Zoom Meeting ${data.id} was created. The user with email "${data.host_email}" is assigned as host.`);
     } else {
-        logger.error(`Zoom - ${response.statusText}`);
+        throw new Error(`Zoom - failed to create meeting with ${response.status} ${response.statusText}`);
```

["fail early, fail often"](https://en.wikipedia.org/wiki/Fail-fast)